Lottery.java

```diff
@@ -0,0 +1,11 @@
+import java.util.Random;
+public class Lottery {
+    public static ball getRandomBall(){
+        Random random = new Random();
+        int numb = random.nextInt(100);
+        ColorSupplier ColorsSupplier = new ColorSupplier();

```

Comment:
The variable name `ColorsSupplier` should start with a lowercase letter to follow Java naming conventions for variables. Consider renaming it to `colorSupplier`.